### ISSUE TITLE
Move files icons to the 'Project Files' node

### ISSUE THREAD

**Issue by skyghis** [markiewb/readmeinprojectview#9]
I think the files should be displayed into the `Project Files` :

![screenshot](https://user-images.githubusercontent.com/1068552/27792131-9230a952-5ff8-11e7-9425-b7d58e9e6eca.png)

This is a Gimp mockup to show possible result.

**Comment by Chris2011** [comment]
Yeah, why not.

**Comment by Chris2011** [comment]
But this is only possible for Maven projects. For normal java we only have "important Files"

**Comment by skyghis** [comment]
IMHO put the files in `Project Files` for Maven projects and in `Important Files` for other projects.

**Comment by Chris2011** [comment]
Yeah, important files also available for PHP and HTML5 projects. Thats good.

**Comment by Chris2011** [comment]
I would prefer to have an option for that which says: Show files under "Project Files/Important Files" where the default is unchecked.

**Comment by skyghis** [comment]
For this Issue I took a look at the Netbeans doc and didn't found how to add elements inside an existing node. @Chris2011 do you know how to do it ?

For the option part it's easy to add it to the existing panel. And I agree, having an option to opt-out this behaviour is a good idea.

**Comment by Chris2011** [comment]
I will check whether I can find a solution or not, In general it is eayy to create your own node and add stuff to it, but here we are talking about an existing one as you pointed out.

**Comment by Chris2011** [comment]
Will not find time for the next 2 weeks (Vacation) Just FYI.

**Comment by nmatt** [comment]
I second this feature request.
